Week3/week3_homework/exercise2.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+function calculateDogAge(puppy_age) {
```

Comment:
Function name is correct but the parameter name is not following naming convention 🙂